src/search-manager/FilterByBlockType.tsx

```diff
@@ -275,14 +277,18 @@ const FilterByBlockType: React.FC<Record<never, never>> = () => {
           value={blockTypesFilter}
         >
           <Menu className="block-type-refinement-menu" style={{ boxShadow: 'none' }}>
+            {
+              // Show applied filter items for block types that are not in the current search results
+              hiddenBlockTypes.map(blockType => <FilterItem key={blockType} blockType={blockType} count={0} />)
```

Comment:
I added this feature to show filtered items not in the current view.
Doing the same for the Tags filter is way more complicated.

If you feel this is not necessary, I can remove it.

Reply:
@rpenido How can we test this manually?

Reply:
Got it. I think it would be better to just hide the whole types filter option in collections tab since it is always empty.

Reply:
The issue is that the filter will still be applied if we hide it. So I think it is better to show it.
We could hide and clear this filter criteria by switching to the Collections tab. But I think the way we are doing now (persisting the filter) is better.

 What do you think @ChrisChV?

Reply:
Can't we hide the filter _and_ not apply it, while we're on the collections tab?